components/butterfly/protocols/swim.proto

```diff
@@ -51,6 +51,7 @@ message Service {
   optional string pkg = 9;
   optional bytes cfg = 10;
   optional SysInfo sys = 12;
+  optional string channel = 13;
```

Comment:
The CensusMember is a starting point for reporting information about a service to EventSrv. It doesn't need to be a 1:1 mapping, though. What I'm saying is that it doesn't make sense to gossip this information with other Supervisors, it just makes sense to ship the information up to the EventSrv. How we build the EventSrv message might need to be refactored a bit to be populated by some additional fields